build-config/conf/crosstool/gcc-4.9.2/uClibc-0.9.33.2/crosstool.armv7a.config

```diff
@@ -259,30 +249,19 @@ CT_BINUTILS_binutils=y
 #
 # GNU binutils
 #
+# CT_BINUTILS_CUSTOM is not set
+CT_BINUTILS_VERSION="2.24"
 # CT_CC_BINUTILS_SHOW_LINARO is not set
-CT_BINUTILS_V_2_25=y
-# CT_BINUTILS_V_2_24 is not set
+# CT_BINUTILS_V_2_26 is not set
+# CT_BINUTILS_V_2_25_1 is not set
+CT_BINUTILS_V_2_24=y
```

Comment:
You commented on https://github.com/opencomputeproject/onie/pull/375 
Why is the binutils version changing for every architecture?

Which is why i made binutils version same across all architectures i.e. 2.24